docs/index.mdx

```diff
@@ -1,26 +1,57 @@
 ---
-title: Saleor Documentation
-slug: /
+title: Saleor docs
```

Comment:
**suggestion (blocking)**: Please put titles in https://en.wikipedia.org/wiki/The_Chicago_Manual_of_Style.

**style (non-blocking)**: I think "Saleor Documentation" is more professional.

Reply:
I started correcting that in navigation, but there are leftovers